conf/appoptics-init.sh

```diff
@@ -50,6 +50,11 @@ if [ "$APPOPTICS_ENABLE_ZOOKEEPER" = "true" ]; then
     mv /opt/appoptics/etc/plugins.d/zookeeper.yaml.example /opt/appoptics/etc/plugins.d/zookeeper.yaml
 fi
 
+if [ "$APPOPTICS_ENABLE_MYSQL" = "true" ]; then
+    mv /opt/appoptics/etc/plugins.d/mysql.yaml.example /opt/appoptics/etc/plugins.d/mysql.yaml
+    sed -i -e "s/root:admin\@tcp(mysql:3306)/$MYSQL_USER:$MYSQL_PASS\@tcp($MYSQL_HOST:$MYSQL_PORT)/" /opt/appoptics/etc/plugins.d/mysql.yaml
```

Comment:
What happens if the user has not set 
MYSQL_HOST
MYSQL_PORT
MYSQL_USER
MYSQL_PASS
?